src/api/ledger/parse/ledger.js

```diff
@@ -13,8 +22,8 @@ function parseTransactions(transactions) {
     return {transactionHashes: transactions};
   }
   return {
-    transactions: _.map(transactions, parseTransaction),
-    rawTransactions: JSON.stringify(transactions)
+    rawTransactions: JSON.stringify(transactions),
```

Comment:
The fact that the ordering matters here is concerning... I think we should just not mutate in the wrapper.

`_.assign({}, transaction, {meta: transaction.metaData})` should work